r/DESCRIPTION

```diff
@@ -0,0 +1,20 @@
+Package: arrow
+Title: Bindings to 'Apache' 'Arrow'
```

Comment:
This package will be a bit more than bindings. Perhaps "R integration with Apache Arrow"?